pkg/api/output.go

```diff
@@ -22,12 +22,12 @@ type Output struct {
 
 // Instance is an instance of the API. This holds configuration for a "run" of Pluto
 type Instance struct {
-	Outputs            []*Output `json:"items,omitempty" yaml:"items,omitempty"`
-	IgnoreDeprecations bool      `json:"-" yaml:"-"`
-	IgnoreRemovals     bool      `json:"-" yaml:"-"`
-	OutputFormat       string    `json:"-" yaml:"-"`
-	ShowAll            bool      `json:"show-all,omitempty" yaml:"show-all,omitempty"`
-	TargetVersion      string    `json:"target-version,omitempty" yaml:"target-version,omitempty"`
+	Outputs            []*Output         `json:"items,omitempty" yaml:"items,omitempty"`
+	IgnoreDeprecations bool              `json:"-" yaml:"-"`
+	IgnoreRemovals     bool              `json:"-" yaml:"-"`
+	OutputFormat       string            `json:"-" yaml:"-"`
+	ShowAll            bool              `json:"show-all,omitempty" yaml:"show-all,omitempty"`
+	TargetVersions     map[string]string `json:"target-versions,omitempty" yaml:"target-versions,omitempty"`
```

Comment:
Since target-versions is always passed via flag, and that flag has a default, we shouldn't be omitting empty here.

```suggestion
	TargetVersions     map[string]string `json:"target-versions" yaml:"target-versions"`
```